lighthouse-core/audits/sized-images.js

```diff
@@ -51,6 +51,29 @@ class SizedImages extends Audit {
     return NON_NEGATIVE_INT_REGEX.test(attr) && !ZERO_REGEX.test(attr);
```

Comment:
I think audit should be `unsized-images`, matching what we are saying is bad. like `unused-javascript`.